src/main/java/com/mraof/minestuck/data/loot_table/MSGiftLootTables.java

```diff
@@ -79,6 +79,7 @@ public void accept(BiConsumer<ResourceLocation, LootTable.Builder> lootProcessor
 						.add(LootItem.lootTableItem(MSItems.MUSIC_DISC_EMISSARY_OF_DANCE.get()).setWeight(1))
 						.add(LootItem.lootTableItem(MSItems.MUSIC_DISC_DANCE_STAB_DANCE.get()).setWeight(1))
 						.add(LootItem.lootTableItem(MSItems.ELECTRIC_AUTOHARP.get()).setWeight(1))
+						.add(LootItem.lootTableItem(MSItems.ANCIENT_THUMB_DRIVE.get()).setWeight(3))
```

Comment:
set the weight to 1 and move it to CONSORT_JUNK_REWARD at the bottom. Its thematically relevant and it will be rarer which is needed to offset how high its grist cost is